app/assets/stylesheets/application.scss

```diff
@@ -12,10 +12,12 @@
  *
  */
 @import "bootstrap-sprockets";
+@import "bootstrap";
```

Comment:
I had thought that we'd want bootstrap-overrides to override bootstrap styling...  :smile:   But yes, the variables was the reason that I'd moved it here.